Documentation/op-guide/grafana.json

```diff
@@ -924,15 +924,15 @@
                     "targets": [{
                         "expr": "etcd_server_leader_changes_seen_total",
                         "intervalFactor": 2,
-                        "legendFormat": "{{instance}} Leader Change Seen",
+                        "legendFormat": "{{instance}} Total Leader Elections",
```

Comment:
I think we can actually change this to Leader Change Seen in one day. and change [1m] to [1day]